cookbook/configuration/pdo_session_storage.rst

```diff
@@ -112,15 +92,74 @@ configuration format of your choice):
             new Reference('pdo'),
-            '%pdo.db_options%',
         ));
         $container->setDefinition('session.handler.pdo', $storageDefinition);
 
-* ``db_table``: The name of the session table in your database
-* ``db_id_col``: The name of the id column in your session table (VARCHAR(128))
-* ``db_data_col``: The name of the value column in your session table (BLOB)
-* ``db_time_col``: The name of the time column in your session table (INTEGER)
-* ``db_lifetime_col``: The name of the lifetime column in your session table (INTEGER)
+Configuring the Table and Column Names
+--------------------------------------
+
+This will expect a ``sessions`` table with a number of different columns.
+The table name, and all of the column names, can be configured by passing
+a second array argument to ``PdoSessionHandler``:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        services:
+            # ...
+            session.handler.pdo:
+                class:     Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
+                arguments:
+                    - "@pdo"
+                    - { 'db_table': 'sessions'}
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <services>
+            <service id="session.handler.pdo"
+                class="Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler">
+                <argument type="service" id="pdo" />
+                <argument type="collection">
+                    <argument key="db_table">sessions</argument>
+                </argument>
+            </service>
+        </services>
+
+    .. code-block:: php
+
+        // app/config/config.php
+
+        use Symfony\Component\DependencyInjection\Definition;
+        // ...
+
+        $storageDefinition = new Definition(
+            'Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler',
+            array(
+                new Reference('pdo'),
+                array('db_table' => 'session')
+            )
+        );
+        $container->setDefinition('session.handler.pdo', $storageDefinition);
+
+.. versionadded:: 2.6
+    The ``db_lifetime_col`` was introduced in Symfony 2.6. Prior to 2.6,
+    this column did not exist.
+
+The following things can be configured:
+
+* ``db_table``: (default ``session``) The name of the session table in your
+  database;
+* ``db_id_col``: (default ``sess_id``) The name of the id column in your
+  session table (VARCHAR(128));
+* ``db_data_col``: (default ``sess_data``) The name of the value column in
+  your session table (BLOB);
+* ``db_time_col``: (default ``sess_time``) The name of the time column in
+  your session table (INTEGER);
+* ``db_lifetime_col``: (default ``sess_lifetime``) The name of the lifetime
+  column in your session table (INTEGER).
 
 Sharing your Database Connection Information
```

Comment:
Boom, love it:

A) Recommend that people use a separate PDO connection (and not to use that connection from in their app)
B) Change things to pass in the credentials directly to PdoSessionHandler (which will take care of the first part, really).

Thanks - these will be good improvements!